datastore/mock_database.go

```diff
@@ -315,6 +316,89 @@ func (mdb *MockDB) CreateOpenidNonce(nonce OpenidNonce) error {
 	return nil
 }
 
+// CreateUserTable mock for creating database User table operation
+func (mdb *MockDB) CreateUserTable() error {
+	return nil
+}
+
+// CreateAccountUserLinkTable mock for creating database AccountUserLink table operation
+func (mdb *MockDB) CreateAccountUserLinkTable() error {
+	return nil
+}
+
+// CreateUser mock for create user operation
+func (mdb *MockDB) CreateUser(user User) error {
+	return nil
+}
+
+// ListUsers mock returning a fixed list of users
+func (mdb *MockDB) ListUsers() ([]User, error) {
+	var users []User
+	users = append(users, User{
+		ID:             1,
+		Username:       "user1",
+		OpenIDIdentity: "https://login.ubuntu.com/+id/Abcyssfmr",
+		Name:           "Rigoberto Picaporte",
+		Email:          "rigoberto.picaporte@ubuntu.com",
+		Role:           Standard})
+	users = append(users, User{
+		ID:             2,
+		Username:       "user2",
+		OpenIDIdentity: "https://login.ubuntu.com/+id/Abcyssfmr",
+		Name:           "Nancy Reagan",
+		Email:          "nancy.reagan@usa.gov",
```

Comment:
It's an honour to have Nancy assist with testing the serial-vault. Does she need a unique OpenID identity?